inc/about.php

```diff
@@ -157,9 +157,13 @@ function cptui_about_page_newsletter() {
 	}
 
 	?>
-	<h3><?php esc_html_e( 'Stay informed', 'custom-post-type-ui' ); ?></h3>
+	<div class='wdsoctosignup'>
+		<?php
+		cptui_newsletter_form();
```

Comment:
Here is a screenshot comparing the old version (on the top) with the new version (on the bottom).

- the old version did not have the white background
- the styling of the old `Subscribe` button was significantly different

![Screen Shot 2020-02-04 at 16 17 38 ](https://user-images.githubusercontent.com/5194588/73787804-02f5f380-476a-11ea-8bed-e682306bc031.png)

Reply:
These design changes were intentional to better match the rest of the page.